config/packages/prod/webpack_encore.yaml

```diff
@@ -0,0 +1,4 @@
+#webpack_encore:
```

Comment:
This file should not exist. If needed, add prod configs in main config file with `when@prod`